<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: url(Slot_Machine_Image) no-repeat;
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="betButton">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>160</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>Bet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pullButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: 2px solid;
border-radius: 25%;
border-color: red;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 20px;
</string>
    </property>
    <property name="text">
     <string>PULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="middleReelLabel">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>170</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;URW Chancery L&quot;, cursive;
font-size: 60px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rightReelLabel">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>170</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;URW Chancery L&quot;, cursive;
font-size: 60px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leftReelLabel">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>170</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;URW Chancery L&quot;, cursive;
font-size: 60px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="betNumberLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="betErrorLabel">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>70</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pullErrorLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="coinsLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="noCoinsLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 50px;</string>
    </property>
    <property name="text">
     <string>Slot Machine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: &quot;Apple Chancery&quot;, cursive;
font-size: 25px;</string>
    </property>
    <property name="text">
     <string>Coins:</string>
    </property>
   </widget>
   <widget class="QLabel" name="coinsWonLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>85</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Coins Won:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
